mail/mlmmj/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           codeberg 1.0
+PortGroup           legacysupport 1.1
+
+codeberg.setup      mlmmj mlmmj RELEASE_1_4_6
```

The correct way to call this is:
```suggestion
codeberg.setup      mlmmj mlmmj 1_4_6 RELEASE_
version             [string map {_ .} ${codeberg.version}]
```
Then you can remove the line that sets `version` below.

---

By default `codeberg.setup` sets up your port to download an automatically-generated gzipped tarball. Projects like mlmmj that offer manually-created tarballs should be changed to use those instead. See:

https://codeberg.org/mlmmj/mlmmj/releases/tag/RELEASE_1_4_6

This project offers a release download called mlmmj-1.4.6.tar.xz which should be used instead because xz compresses better than gzip and because it likely has already had autoreconf run in it so that if a lack of configure script is the only reason why you were running autoreconf you can then stop doing that.

Getting this release tarball instead of the automatically-generated one requires setting `use_xz yes`. The `checksums` will also change. Unfortunately, unlike the github portgroup upon which it is based, the codeberg portgroup does not have a feature to change `master_sites` to use a release download so you'll either have to study the github and codeberg portgroups and enhance the codeberg portgroup to add this feature (in the github portgroup it's called `github.tarball_from`) or you'll have to override `master_sites` with the release download URL as you originally did before adding the codeberg portgroup.

---

I'll try this method instead of autoconf.